app/src/main/java/ch/epfl/culturequest/social/Profile.java

```diff
@@ -50,17 +53,19 @@ public Profile(String username, String profilePicture) {
         this.profilePicture = profilePicture;
         this.images = List.of();
         this.score = 0;
+        this.Friends = new ArrayList<>();
     }
 
-    public Profile(String uid, String name, String username, String email, String phoneNumber, String profilePicture, List<Image> images, Integer score) {
+    public Profile(String uid, String name, String username, String email, String phoneNumber, String profilePicture, ArrayList<String> friends, Integer score) {
```

Comment:
why did you remove the images attribute in the constructor?

Reply:
Because the Profile won't have images anymore since it has been replaced by posts